packages/mail/script/generateTextTemplates.js

```diff
@@ -30,13 +31,59 @@ const run = async () => {
       await fs.writeFile(
         file.replace(/\.html$/gi, '.txt'),
         htmlToText
-          .fromString(html, { ignoreImage: true })
-          .replace(/^ /gm, '')
-          .replace(/kontakt@republik\.ch \[kontakt@republik\.ch\]/gm, 'kontakt@republik.ch'),
```

Comment:
this is not needed anymore?

Reply:
it's handled by `toHideSameLink` (aka `hideLinkHrefIfSameAsText` option which is baked into our custom format anchor)

Reply:
yes, yes, after understanding the first question, I now also see this.